src/const/urls.js

```diff
@@ -0,0 +1,6 @@
+const urls = {
+  dogNinjaApi: `https://api.api-ninjas.com/v1/dogs?min_height=6`,
+  nameApi: "https://randomuser.me/api/?results=22&nat=us&inc=name",
+};
```

Comment:
all those URls shouldn't contain params

Reply:
it will be your base for all the API calls so it needs to have default state

Reply:
In order to get a default result for the dog API, the documentation states that at least 1 of the provided parameters needed to be used. I experimented by requesting a "results=20" in the parameter, however, it resulted in an error.  Using "min_height=6" will give me any dog that is at least 6 inches in height as an adult. However, I will change it down to at least 4 inches according to the world record for the shortest dog breed. 

As far as the name API, Since I am only requesting it once at the beginning of the function call I don't need to modify the parameters as of now. 

These URLs are currently my default results that should always be called when they are requested.

Reply:
It is all valid, but we are creating those "default state URLs" in case you have multiple environments to manage such as testing, staging, and production. (so it is easier to switch it)
From that We can create functions that will provide URLs with params...and It would be good to include in the comments your experiment results, so everyone knows why you chose it.

It is in general good practice after fixing a bug that is unconventional to comment either by yourself or just link to the StackOverflow solution from where you took it

Reply:
I changed the URLs and then moved those params into the Axios request so that way people can see what params are needed.